modules/ROOT/pages/provisioning-virtualbox.adoc

```diff
@@ -4,193 +4,144 @@ This guide shows how to provision new Fedora CoreOS (FCOS) nodes on the VirtualB
+
 [source, bash]
 ----
-vboxmanage createmedium --filename "${VM_DISK}" --size 10240 --format VDI
-vboxmanage storageattach "${VM_NAME}" --storagectl SATA --type hdd --port 0 --device 0 --medium "${VM_DISK}"
+IGN_PATH="/path/to/config.ign"
+VM_NAME=my-instance
+VBoxManage guestproperty set "$VM_NAME" /Ignition/Config "$(cat $IGN_PATH)"
 ----
 
-[NOTE]
-====
-*Fedora CoreOS requires the root filesystem to be at least 8 GiB.* For practical reasons, disk images for some platforms ship with a smaller root filesystem, which by default automatically expands to fill all available disk space. If you add additional partitions after the root filesystem, you must make sure to explicitly resize the root partition as shown so that it is at least 8 GiB.
+==== Ignition config size limitations
 
-Currently, if the root filesystem is smaller than 8 GiB, a warning is emitted on login. Starting from June 2021, if the root filesystem is smaller than 8 GiB and is followed by another partition, Fedora CoreOS will refuse to boot. For more details, see https://github.com/coreos/fedora-coreos-tracker/issues/586[this bug].
-====
+The length of the `/Ignition/Config` guestinfo property is constrained by the maximum length of a command line in your host operating system. The OS-specific limits are approximately:
 
-- Attach a DVD drive and the live ISO to the VM:
+[cols="1,1"]
+|===
+|OS
+|Limit
 
-[source,bash,subs="attributes"]
-----
-FCOS_DISK=$(realpath ~/Downloads/fedora-coreos-{stable-version}-live.x86_64.iso)
+|Linux
+|128 KiB
+|macOS
+|256 KiB
+|Windows shells
+|8 KiB
+|===
 
-vboxmanage storagectl "$\{VM_NAME}" --name IDE --add ide --controller PIIX4 --hostiocache on --portcount 2 --bootable on
-vboxmanage storageattach "$\{VM_NAME}" --storagectl IDE --type dvddrive --port 1 --device 0 --medium "$\{FCOS_DISK}"
-----
-
-You can now boot the target system you have configured. The ISO is capable of bringing up a fully functioning FCOS system purely from memory (i.e. without using any disk storage). Once booted, you will have access to a bash command prompt.
+If your Ignition config is larger than this limit, you can host the config on an HTTPS server and refer to it from a small _pointer config_, as follows:
 
-You can now run `coreos-installer` inside the VM:
-[source, bash]
+. Upload your Ignition config to an HTTPS server.
+. xref:remote-ign.adoc[Create a Butane pointer config] that specifies the URL of your full Ignition config:
++
+[source, yaml]
 ----
-sudo coreos-installer install /dev/sda \
-    --insecure-ignition --ignition-url http://10.0.2.2:8000/example.ign
+variant: fcos
+version: 1.4.0
+ignition:
+  config:
+    replace:
+      source: https://example.com/config.ign
 ----
+. Use xref:producing-ign.adoc[Butane] to convert the Butane config to an Ignition config.
+. Set the `/Ignition/Config` guest property to the contents of the pointer Ignition config, following the instructions in <_setting_the_ignition_config>.
```

Comment:
Ah, [looks](https://docs.asciidoctor.org/asciidoc/latest/macros/xref/#internal-cross-references) like the xref should use double angle brackets.

Reply:
Ah okay resolved I think?